introduction.md

```diff
@@ -0,0 +1,3 @@
+I'm Felipe Mota I'm a iOS and Android developer with +3 years experience mobile.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.